core/account.js

```diff
@@ -71,6 +82,47 @@ Account.prototype.tx = function tx(to, value, data='0x00') {
     return '0x' + tx.serialize().toString('hex');
 };
 
+/**
+ *
+ * Signing message by account
+ *
+ * @param {String} message Text message to sign
+ * @return {Object} Message
+ */
+Account.prototype.signMessage = function signMessage(message) {
+    let signedMessage = {};
+
+    signedMessage.publicKey = this.publicKey;
+    signedMessage.message = message;
+
+    let hash = keccak256(Buffer.from(message));
+    signedMessage.signature = secp256k1.sign(hash, this.secretKey).signature;
+
+    return signedMessage;
+}
+
+/**
+ *
+ * Verifying message signed by account
+ *
+ * @param {Object} Message object signed by signMessage function
+ * @param {Buffer} Public key if signed message doesn't contain public key or need another public key
+ *
+ * @return {Boolean} Result of signature verification
+ */
+Account.verifyMessage = function verifyMessage(signedMessage, publicKey) {
```

Comment:
What if it's not a signedMessage but just `(message, pk)` there should be a mutation here or arguments variation. Otherwise this makes no sense unless you force others to follow `signedMessage` type.